tests/openshift/openshift-tests.yml

```diff
@@ -1,9 +1,37 @@
----
+--- 
 - hosts: localhost
   tasks:
+  
   - name: Test cluster health reports OK
     uri:
-      url: https://ocp.customer2.cor00005.cna.ukcloud.com:8443/healthz/ping
+      url: "{{server}}/healthz/ping"
       status_code: 200
       body: ok
       validate_certs: no
+  
+  - name: Request URL to confirm homepage works
+    uri:
+      url: "{{server}}"
+      status_code: 200
+      validate_certs: no
+
+  - name: Login to openshift
+    command: oc login "{{server}}" -u "{{OPENSHIFT_USERNAME}}" -p "{{OPENSHIFT_PASSWORD}}" --insecure-skip-tls-verify
+
+  - name: Creating project
```

Comment:
please be consistent in your names. As this is a test, it's OK to say "Create Project" (rather than "Ensure project is created")